xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamNavigator.cpp

```diff
@@ -1535,6 +1535,8 @@ void CDVDInputStreamNavigator::GetVideoResolution(uint32_t* width, uint32_t* hei
   if (!m_dvdnav) return;
 
   dvdnav_status_t status = m_dll.dvdnav_get_video_resolution(m_dvdnav, width, height);
+  if (status != DVDNAV_STATUS_OK)
+    width = height = 0;
```

Comment:
A well, you are right. Today is not my day. I should (and will) stop coding for now.